components/SegmentList.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <section>
+    <base-headline
+      v-if="title"
```

Comment:
now I'm pretty sure this is super duplicated v-if. Does the component itself not already have a v-if inside?